src/types.ts

```diff
@@ -1307,9 +1315,9 @@ export class ZodString extends ZodType<string, ZodStringDef, string> {
   get isTime() {
     return !!this._def.checks.find((ch) => ch.kind === "time");
   }
-  get isDuration() {
-    return !!this._def.checks.find((ch) => ch.kind === "duration");
-  }
+  // get isDuration() {
+  //   return !!this._def.checks.find((ch) => ch.kind === "duration");
+  // }
```

Comment:
nit: remove commented methods entirely

Reply:
Keeping it could possibly simplify merging upstream changes in the future. We can [remove them during TS transpilation](https://www.typescriptlang.org/tsconfig/#removeComments), though.